changeset:   167909:f9e2a6254a19
tag:         tip
user:        Solomon Chiu <schiu@mozilla.com>
date:        Wed Feb 12 01:36:45 2014 +0800
summary:     Rename KK's file to corrensponding Gonk version

diff --git a/widget/gonk/nativewindow/BufferQueue.cpp b/widget/gonk/nativewindow/GonkBufferQueueKK.cpp
rename from widget/gonk/nativewindow/BufferQueue.cpp
rename to widget/gonk/nativewindow/GonkBufferQueueKK.cpp
diff --git a/widget/gonk/nativewindow/BufferQueue.h b/widget/gonk/nativewindow/GonkBufferQueueKK.h
rename from widget/gonk/nativewindow/BufferQueue.h
rename to widget/gonk/nativewindow/GonkBufferQueueKK.h
diff --git a/widget/gonk/nativewindow/ConsumerBase.cpp b/widget/gonk/nativewindow/GonkConsumerBaseKK.cpp
rename from widget/gonk/nativewindow/ConsumerBase.cpp
rename to widget/gonk/nativewindow/GonkConsumerBaseKK.cpp
diff --git a/widget/gonk/nativewindow/ConsumerBase.h b/widget/gonk/nativewindow/GonkConsumerBaseKK.h
rename from widget/gonk/nativewindow/ConsumerBase.h
rename to widget/gonk/nativewindow/GonkConsumerBaseKK.h
diff --git a/widget/gonk/nativewindow/Surface.cpp b/widget/gonk/nativewindow/GonkNativeWindowClientKK.cpp
rename from widget/gonk/nativewindow/Surface.cpp
rename to widget/gonk/nativewindow/GonkNativeWindowClientKK.cpp
diff --git a/widget/gonk/nativewindow/Surface.h b/widget/gonk/nativewindow/GonkNativeWindowClientKK.h
rename from widget/gonk/nativewindow/Surface.h
rename to widget/gonk/nativewindow/GonkNativeWindowClientKK.h
diff --git a/widget/gonk/nativewindow/BufferItemConsumer.cpp b/widget/gonk/nativewindow/GonkNativeWindowKK.cpp
rename from widget/gonk/nativewindow/BufferItemConsumer.cpp
rename to widget/gonk/nativewindow/GonkNativeWindowKK.cpp
diff --git a/widget/gonk/nativewindow/BufferItemConsumer.h b/widget/gonk/nativewindow/GonkNativeWindowKK.h
rename from widget/gonk/nativewindow/BufferItemConsumer.h
rename to widget/gonk/nativewindow/GonkNativeWindowKK.h
diff --git a/widget/gonk/nativewindow/IGraphicBufferConsumer.cpp b/widget/gonk/nativewindow/IGonkGraphicBufferConsumer.cpp
rename from widget/gonk/nativewindow/IGraphicBufferConsumer.cpp
rename to widget/gonk/nativewindow/IGonkGraphicBufferConsumer.cpp
diff --git a/widget/gonk/nativewindow/IGraphicBufferConsumer.h b/widget/gonk/nativewindow/IGonkGraphicBufferConsumer.h
rename from widget/gonk/nativewindow/IGraphicBufferConsumer.h
rename to widget/gonk/nativewindow/IGonkGraphicBufferConsumer.h

